app/src/main/java/org/simple/clinic/login/pin/LoginPinEffectHandler.kt

```diff
@@ -88,7 +88,8 @@ class LoginPinEffectHandler @AssistedInject constructor(
         updatedAt = entry.updatedAt!!,
         loggedInStatus = User.LoggedInStatus.OTP_REQUESTED,
         registrationFacilityUuid = entry.registrationFacilityUuid!!,
-        currentFacilityUuid = entry.registrationFacilityUuid
+        currentFacilityUuid = entry.registrationFacilityUuid,
+        teleconsultPhoneNumber = null
```

Comment:
`OngoingLoginEntry` is created from the `UsersApi#activate` API endpoint which returns a complete `LoggedInUserPayload` which will include the teleconsultation number.

So setting it to `null` is the incorrect thing to do here since we should also update `OngoingLoginEntry` to save the teleconsult number.

Reply:
Ah okay, my bad. I was under the impression that `OngoingLoginEntry` is only local to device for registration purpose.